examples/examples-cpu/nyc-taxi-snowflake/xgboost-dask.ipynb

```diff
@@ -27,8 +25,27 @@
    "outputs": [],
    "source": [
     "import os\n",
-    "import numpy as np\n",
+    "import warnings\n",
+    "\n",
+    "import cloudpickle\n",
+    "import dask.dataframe as dd\n",
+    "import dask_xgboost\n",
+    "import snowflake.connector\n",
+    "\n",
+    "from dask import delayed\n",
+    "from dask.distributed import Client, wait\n",
+    "from dask_ml.metrics import mean_squared_error\n",
+    "from dask_saturn import SaturnCluster\n",
     "\n",
+    "warnings.simplefilter(\"ignore\")"
```

Comment:
I think we should avoid this and instead try to track down the source of any warnings.